### ISSUE TITLE
Reconcile console and TF Okta configuration

### ISSUE THREAD

**Issue by jdorothy** [CDCgov/prime-simplereport#2080]
Our Terraform Okta config isn't quite in sync with our actual Okta config, and we need to bring the Terraform config inline before we can start running `ops/*/persistent` in the pipeline.

**Comment by jdorothy** [comment]
On hold until we find out what causes `terraform apply` to behave differently in production.

**Comment by jdorothy** [comment]
Opened a case with Okta support, and was asked to file a GH issue: https://github.com/okta/terraform-provider-okta/issues/564

**Comment by jdorothy** [comment]
Bug fixed!
